src/definitions/behaviors/form.js

```diff
@@ -1801,10 +1802,16 @@
         rules: {
 
             // is not empty or blank string
-            empty: function (value) {
+            notEmpty: function (value) {
                 return !(value === undefined || value === '' || (Array.isArray(value) && value.length === 0));
             },
 
+            empty: function (value, ancillary, module) {
+                module.debug('Rule "empty" is deprecated and will be removed in a future version. Use the "notEmpty" rule instead.');
```

Comment:
not many developers have debug verbosity enabled, IMHO the lowest visible level by default should be used

Reply:
Nevermind: I moved the check to the settings builder so it will be warned into the console on instantiation once.

Please review again

Reply:
Yes, this looks good!